project/MicroServiceBuild.scala

```diff
@@ -8,6 +8,7 @@ import uk.gov.hmrc.versioning.SbtGitVersioning
 object MicroServiceBuild extends Build with MicroService {
 
   val appName = "help-to-save-stub"
+  private val mockitoAllVersion = "1.10.19"
```

Comment:
we're using `scalamock` in the other projects - for consistency is it not a good idea to use the saame?